src/igvfd/tests/fixtures/schemas/in_vitro_system.py

```diff
@@ -75,3 +75,13 @@ def in_vitro_system_v3(in_vitro_cell_line):
         'accession': 'IGVFSM222IIV'
     })
     return item
+
+
+@pytest.fixture
+def in_vitro_system_v4(in_vitro_organoid):
+    item = in_vitro_organoid.copy()
+    item.update({
+        'schema_version': '5',
```

Comment:
I think it should be 4 and not 5?

Reply:
Does that mean  i have to change the schema_version in src/igvfd/schemas/in_vitro_system.json as well?
```
"properties": {
        "schema_version": {
            "default": "5"
```

Reply:
you create in_vitro_system_v4, then for some reason change the schema version of it to 5? I am not following why?